Readme.md

```diff
@@ -432,8 +432,8 @@ aecu.contentUpgradeBuilder()
         .forChildResourcesOf("/content/we-retail/ca/en")
         .filterByNodeName("jcr:content")
         .doRename("newNodeName")
-        .doCopyResourceToRelativePath("subNode")
-        .doCopyResourceToRelativePath("../subNode")
+        .doCopyResourceToRelativePath("subNode", "newName")
```

Comment:
New name should be optional. You can simply expose two methods in API and call with null internally.

Reply:
good point